marimbabot_vision/scripts/data_augmentation.py

```diff
@@ -26,16 +26,17 @@ def augment_sample(i):
 
     # set of simpler transformations
     transformations = dict({
-        "random_gamma": [A.RandomGamma(always_apply=True)], 
-        "random_contrast": [A.RandomBrightnessContrast(always_apply=True)],
         "blur": [A.Blur(blur_limit=15, always_apply=True)],
-        "median_blur" : [A.MedianBlur(blur_limit=15, always_apply=True)],
-        "glass_blur": [A.GlassBlur(always_apply=True)],
+        "median_blur" : [A.MedianBlur(blur_limit=5, always_apply=True)],
+        "glass_blur": [A.GlassBlur(sigma=0.4,always_apply=True)],
         "zoom_blur": [A.ZoomBlur(always_apply=True)],
-        "image_compression": [A.ImageCompression(quality_lower=20 ,always_apply=True)],
-        "downscale": [A.Downscale(scale_min=0.5, scale_max=0.5, always_apply=True, interpolation=cv2.INTER_NEAREST)],
+        "image_compression": [A.ImageCompression(quality_lower=60, quality_upper=60 ,always_apply=True)],
+        "downscale": [A.Downscale(scale_min=0.4, scale_max=0.4, always_apply=True, interpolation=cv2.INTER_NEAREST)],
```

Comment:
Sane here. Why are min and max the same?

Reply:
you are right